libs/site-base/src/containers/AddAccount/FromJson/index.tsx

```diff
@@ -14,29 +16,24 @@ const translations = defineMessages({
       description: 'app.account.restore.createAccount.button: The button to redirect to the create an account page for the restore your account page'}
 });
 
-function readFile(file: File): Promise<ReadFileData> {
-  return new Promise((resolve, reject) => {
-    const reader = new FileReader();
-    reader.onload = (e) => {
-      const target = e.target;
-      resolve({
-        wallet: target?.result as string,
-        name: undefined
-      });
-    };
-    reader.onerror = reject;
-    reader.readAsText(file);
-  });
-}
+export const FromJson = () => {
```

Comment:
Moved the file reading bit into the FileUpload dlg.  No idea why it was split out here originally